### ISSUE TITLE
Feature Request: Unread messages sorted by feed

### ISSUE THREAD

**Issue by Twilek-de** [FreshRSS/FreshRSS#2060]
I have just migrated to FreshRSS from TinytinyRSS because of this (https://flameeyes.blog/2017/09/03/tiny-tiny-rss-dont-support-nazi-sympathisers/) and I have to say I am quite impressed. Kudos to @Alkarex for the excellent support here on Github!

There is just one thing I am still missing (or haven´t figured out how to do), which TT could do. Namely to show the yet unread messages sorted by feed. So you had let´s say all unread Engadget posts under a heading, then all Slashdot posts and so on (each sorted by date). I know that I could archive something similar by clicking throught all sites on the sidebar one after another but that is quite a hassle. Sorting by feed gave a very structured and quick overview as it made it easy to focus on different sites (I think everybody has some high volume feeds where you just skim the headlines and other specific feeds where every post counts). Having something similar would be quite cool.

**Comment by Alkarex** [comment]
I am always glad to see people moving to FreshRSS for idealogical reasons (which play a central role in FreshRSS), and I hope you will stay *also* for its technical merits :-)
Although smaller than TT-RSS, we do spend some efforts in ensuring that the community here is more friendly and polite.

Regarding caching (which we have been using and promoting for a long time), here is a related recent comment https://github.com/FreshRSS/FreshRSS/issues/1020#issuecomment-430392157

Grouping by feeds in the same view is not possible at the moment BUT when you mark a feed as read, you are (by default) automatically presented with the next feed. It is also possible to customize various keyboard shortcuts to jump between feeds.

**Comment by Twilek-de** [comment]
Yup the jumping to the next feed works quite well. I still prefer all feeds in one list but this isn´t a show stopper just a minor quirk.

**Comment by Alkarex** [comment]
This is possible since 1.27.0 thanks to:
* https://github.com/FreshRSS/FreshRSS/pull/7702
